src/wallet/wallet.cpp

```diff
@@ -4453,4 +4477,17 @@ std::optional<CKey> CWallet::GetKey(const CKeyID& keyid) const
     }
     return std::nullopt;
 }
+
+void CWallet::WriteBestBlock() const
+{
+    AssertLockHeld(cs_wallet);
+
+    if (!m_last_block_processed.IsNull()) {
+        CBlockLocator loc;
+        chain().findBlock(m_last_block_processed, FoundBlock().locator(loc));
+
+        WalletBatch batch(GetDatabase());
+        batch.WriteBestBlock(loc);
+    }
+}
```

Comment:
Now that we have this `WriteBestBlock` function, something similar happens within `BackupWallet` as well: https://github.com/bitcoin/bitcoin/blob/8309a9747a8df96517970841b3648937d05939a3/src/wallet/wallet.cpp#L3267-L3274

Do you think we can leverage `WriteBestBlock` over there? I mention because the pattern is quite similar and I would prefer to have the same function being called.

Reply:
Indeed, done.